site/docs/samples/notebook.md

```diff
@@ -1,162 +1,239 @@
-    For more information on the content please see the comments in [third_pary/egeria/usage/create_new_asset.sh](https://github.com/ibm/the-mesh-for-data/blob/master//third_party/egeria/usage/create_new_asset.sh) for more details.
-1. Register the dataset in the catalog
-
-    - Setup port forwarding for communicating with Egeria
-
-        ```bash
-        cd third_party/egeria/usage
-        kubectl port-forward -n egeria-catalog svc/lab-core 9443:9443 &
-        ```
-    - Wait for the port-forward to take effect.
-
-    - Register the dataset in the catalog with the tag 'finance'.
+    For experimentation you can install MinIO to your cluster instead of using a cloud service.
     
-        ```bash
-        ./create_new_asset.sh ../../../samples/kubeflow/example_transactions.csv.json 'finance'
-        ```
-    - Cleanup the port forwarding using the following.
+    1. Install Minio to the currently active namespace:
+      ```bash
+      helm repo add minio https://helm.min.io
+      helm install --wait minio minio/minio
+      ```
+    1. Create a port-forward to connect to MinIO UI:
+      ```bash
+      kubectl port-forward svc/minio 9000 &
+      ```
+    1. Open [http://localhost:9000](http://localhost:9000) and login with the printed access key and secret key:
+      ```
+      ACCESS_KEY=$(kubectl get secret minio -o jsonpath="{.data.accesskey}" | base64 --decode)
+      SECRET_KEY=$(kubectl get secret minio -o jsonpath="{.data.secretkey}" | base64 --decode)
+      echo "Access Key: ${ACCESS_KEY}"
+      echo "Secret Key: ${SECRET_KEY}"
+      ```
+    1. Click the :fontawesome-solid-plus-circle: button in the bottom right corner and then **Create bucket** to create a bucket (e.g. "demo").
+    1. Click the :fontawesome-solid-plus-circle: button again and then **Upload files** to upload the CSV file to the newly created bucket.
     
+## Register the dataset in a data catalog
+
+Register the credentials required for accessing the dataset. Replace the values for `access_key` and `secret_key` with the values from the object storage service that you used and run:
+
+```yaml
+cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: data-csv
+type: Opaque
+stringData:
+  access_key: "${ACCESS_KEY}"
+  secret_key: "${SECRET_KEY}"
+EOF
+```
+
+Then, register the data asset itself in the catalog. Replace the values for `endpoint`, `bucket` and `objectKey` with values from the object storage service that you used and run:
+
+```yaml
+cat << EOF | kubectl apply -f -
+apiVersion: katalog.m4d.ibm.com/v1alpha1
+kind: Asset
+metadata:
+  name: data-csv
+spec:
+  secretRef: 
+    name: data-csv
+  assetDetails:
+    dataFormat: csv
+    connection:
+      type: s3
+      s3:
+        endpoint: "http://minio.m4d-notebook-sample.svc.cluster.local:9000"
+        bucket: "demo"
+        objectKey: "data.csv"
+  assetMetadata:
+    geography: us-south
```

Comment:
```suggestion
    geography: theshire
```

Reply:
This change is required to avoid having the need to copy the data (cluster-metadata config map has "theshire" value as default value for Region)